roles/prerequisites/tasks/main.yaml

```diff
@@ -14,23 +14,31 @@
   stat:
     path: "{{ workdir.vm_data }}/{{ base_image_name }}"
   register: result 
+  tags:
+    - download_images
```

Comment:
I'm not sure why this is needed, the algorithm was already intelligent enough to not download an image twice if it already existed.

Reply:
I create an offline branch which uses these and puts them in ~/usb/

Reply:
I don't understand why this is needed, can you remove the whole tag thing from this PR? I'd be happy to accept the other suggestions.